docroot/themes/humsci/humsci_basic/src/scss/settings/_variables.general.scss

```diff
@@ -13,19 +13,24 @@ $hb-grid-media: (
 ) !default;
 
 $hb-screen-margins: (
-  'xs': 1.75rem,
-  'sm': 2rem,
-  'md': 2.25rem,
-  'lg': 2.5rem,
-  'xl': 3rem,
-  '2xl': 3.5rem,
+  'xs': 2.8rem,
+  'sm': 3.2rem,
+  'md': 3.6rem,
+  'lg': 4rem,
+  'xl': 4.8rem,
+  '2xl': 5.6rem,
 ) !default;
 
-$hb-gutter-width: 3rem;
-$hb-layout-max-width: 75rem;
+$hb-gutter-width: 4.8rem;
```

Comment:
Looking at all of the rem values in the variables files if/when we update to use a base font fo 16px all of these rems will need to be adjusted again. Instead we should probably convert these to pixel values and then wrap the variable in the calculate rems function. Let me know if that doesn't make sense and we can pair on it.